.gitignore

```diff
@@ -47,3 +47,6 @@ amplifytools.xcconfig
 .secret-*
 **.sample
 #amplify-do-not-edit-end
+
+# Local Netlify folder
+.netlify
```

Comment:
I thought we're not doing Netlify anymore. Should we still have this here?

Reply:
Yes, we don't need it anymore.

Reply:
Yes, we are not doing it anymore>

Reply:
@yotlaire please remove netlify related in this PR. Thank you!